compiler/rustc_index/src/bit_set.rs

```diff
@@ -234,6 +234,28 @@ impl<T: Idx> DenseBitSet<T> {
         self.clear_excess_bits();
     }
 
+    /// Checks whether any bit in the given range is a 1.
+    #[inline]
+    pub fn contains_any(&self, elems: impl RangeBounds<T>) -> bool {
+        let Some((start, end)) = inclusive_start_end(elems, self.domain_size) else {
+            return true;
```

Comment:
Shouldn't this be `false`?

Reply:
I guess it depends on how we count bits outside bounds. I thought I recalled seeing another bitset type assume that out-of-bounds bits are 1s, but on trying to double-check that I can't find the source so I might just be misremembering. In my usecase it's safer to treat it that way, but I'd be okay with changing this to returning an `Option<bool>` instead with this being the `None` case.

Though, the docs for `DenseBitSet` do say "All operations that involve an element will panic if the element is equal to or greater than the domain size" which could also be interpreted as unwrapping here.

Reply:
Ah, I misread the code for `inclusive_start_end`. You're right, sorry!